experimental/PropertyDDS/packages/property-binder/src/internal/property_element.js

```diff
@@ -301,16 +301,17 @@ class PropertyElement {
    *
    * @param {Array.<String>|String} in_child - the tokenized path, or single child
    * @param {Object} in_options - parameter object
+   * @param {Array.<PathHelper.Token_Types>|undefined} in_tokensTypes - The types of tokens if in_child is tokenized path.
    * @param {Object.<{referenceResolutionMode:LYNX.Property.BaseProperty.REFERENCE_RESOLUTION}>} }
    *    [in_options.referenceResolutionMode=ALWAYS] How should this function behave
    *    during reference resolution?
    *
    * @return {PropertyElement} the element representing the child. If it does not exist,
    *   this.isValid() will return false
    */
-  getChild(in_child, in_options = RESOLVE_ALWAYS) {
+  getChild(in_child, in_options = RESOLVE_ALWAYS, in_tokensTypes) {
```

Comment:
It feels more correct to add the parameter directly after in_child, but it will require more code changes. 
@ruiterr  if you think it still worth the afford, I can make the necessary changes, but since it's used internally, it shouldn't make much difference.